app/controllers/application.js

```diff
@@ -91,7 +93,7 @@ export default class ApplicationController extends ParachuteController {
 
   @computed('lat', 'lng')
   get center() {
-    return [this.get('lat'), this.get('lng')];
+    return [this.get('lng'), this.get('lat')];
```

Comment:
Is this fixing a bug you noticed along the way? We've had issues keeping these straight in the app in the past.

Reply:
Yes. The behavior was noticeable when the application first loaded. The viewport would be somewhere in the Atlantic and then jump to lower Manhattan. I believe it was trying to start in Africa but was clamped to the local water by the map's bounds.